tests/Block/PageListBlockServiceTest.php

```diff
@@ -70,18 +70,9 @@ public function testExecute(): void
             'template' => '@SonataPage/Block/block_pagelist.html.twig',
         ]);
 
-        $blockService = new PageListBlockService('block.service', $this->templating, $this->pageManager);
-        $blockService->execute($blockContext);
+        $blockService = new PageListBlockService('block.service', $this->twig, $this->pageManager);
+        $response = $blockService->execute($blockContext);
 
-        static::assertSame('@SonataPage/Block/block_pagelist.html.twig', $this->templating->view);
-
-        static::assertSame($blockContext, $this->templating->parameters['context']);
-        static::assertIsArray($this->templating->parameters['settings']);
-        static::assertInstanceOf(BlockInterface::class, $this->templating->parameters['block']);
-        static::assertCount(2, $this->templating->parameters['elements']);
-        static::assertContains($page1, $this->templating->parameters['elements']);
-        static::assertContains($page2, $this->templating->parameters['elements']);
-        static::assertCount(1, $this->templating->parameters['systemElements']);
-        static::assertContains($systemPage, $this->templating->parameters['systemElements']);
```

Comment:
to replace `EngineInterface` to `Twig\Environment`, do you need to remove all those asserts?